src/KM_Controls.pas

```diff
@@ -28,7 +28,7 @@   TKMPanel = class;
   { TKMMaster }
   TKMMasterControl = class
   private
-    fCtrl: TKMPanel; //Parentmost control (TKMPanel with all its childs)
+    fMasterPanel: TKMPanel; //Parentmost control (TKMPanel with all its childs)

```

Comment:
1. Renamed TKMMasterControl.fCtrl into TKMMasterControl.fMasterPanel as it means exactly master panel (main panel)